pkg/controllers/dynakube/proxy/reconciler.go

```diff
@@ -105,18 +106,19 @@ func (r *Reconciler) createProxyMap(ctx context.Context, dynakube *dynatracev1be
 		portField:     []byte(port),
 		usernameField: []byte(username),
 		passwordField: []byte(password),
+		schemeField:   []byte(scheme),
 	}, nil
 }
 
-func parseProxyUrl(proxy string) (host, port, username, password string, err error) { //nolint:revive // maximum number of return results per function exceeded; max 3 but got 5
+func parseProxyUrl(proxy string) (scheme, host, port, username, password string, err error) { //nolint:revive // maximum number of return results per function exceeded; max 3 but got 6
```

Comment:
Do we even need this function? It doesn't really have any special logic.

Reply:
Not really, but it doesn't hurt either IMO